core/main.go

```diff
@@ -51,7 +51,7 @@ func main() {
 func run(client *cmd.Client, args ...string) {
 	app := cmd.NewApp(client)
 	if err := app.Run(args); err != nil {
-		log.Fatal("Error running app:", err)
+		log.Fatalf("Error running app: %v\n", err)
```

Comment:
Nit: can we standardize error passing w) k,v pairs over formated strings?

Reply:
Do you have an example?
This happens to be one of the few exceptional cases where we don't have our normal logger.

Reply:
(and the only reason I touched it is because it was surprisingly not inserting a space between the two, which confusing printed `Error running app:toml: ...`)

Reply:
This is probably something we can do outside of this PR but I feel like log.Fatalw("some message", "err", err) should be our standard (or that we should have some standard indicating the underlying error.

Reply:
My bad, just noticed we don't have our logger here and this is the std logger :man_facepalming: